### DISCUSSION TITLE
Build failed_Assignment to an import specifier is not allowed

### DISCUSSION THREAD

**Discussion by IrynaBondarenko7** [parcel-bundler/parcel#9018] [Q&A] [answered]
Hello, I need help. I have Build failed
@parcel/transformer-js: Assignment to an import specifier is not allowed

  /home/runner/work/masseur_website/masseur_website/src/js/handlers.js:[26](https://github.com/IrynaBondarenko7/masseur_website/actions/runs/4992899121/jobs/8941248986#step:3:27):3
  >  1 | import { formData, localStorageKeys, sendContactData } from './form';
  >    |          ^^^^^^^^ Originally imported here
     2 | import { onCloseModal, onModalOpen } from './modal';
     3 | import { refs } from './refs';
     4 | 
     5 | export function onOpenModalBtnClick() {
     6 |   onModalOpen();
     7 | }
     8 | export function onCloSeModalButtonClick() {
     9 |   onCloseModal();
    10 | }
    11 | 
    12 | export function onFormSubmit(evt) {

Error: Process completed with exit code 1.

I use the named export, I don't understand what the problem is, all the paths are written correctly

**Comment by mischnic** [comment, marked as answer]
The error message means that you do `formData = ...;` or `localStorageKeys = ...;` or `sendContactData = ...;` somewhere in that file. That should be removed

**Reply by IrynaBondarenko7** [reply]
wow it helped, thank you very much, I would not have found a solution myself!)))
